src/test/java/com/browserstack/SingleTest.java

```diff
@@ -5,17 +5,28 @@
 
 import org.testng.Assert;
 import org.testng.annotations.Test;
+import org.openqa.selenium.JavascriptExecutor;
 
 public class SingleTest extends BrowserStackTestNGTest {
 
     @Test
     public void test() throws Exception {
         driver.get("https://www.google.com/ncr");
+        try {
+            WebElement ele = driver.findElement(By.xpath("//div[text()='I agree']"));
+            if(ele.isDisplayed()){
+                ((JavascriptExecutor)driver).executeScript("arguments[0].scrollIntoView()", ele);
+                Thread.sleep(5000);
+                ele.click();
+            }
+        }
+        catch(Exception e){
+            System.out.print(e);
```

Comment:
Will this print the actual message or just tell the object type is of exception class?

Reply:
It will print this kind of message
```org.openqa.selenium.NoSuchElementException: no such element: Unable to locate element: {"method":"xpath","selector":"//div[text()='I agree']"}```